util/chplenv/__init__.py

```diff
@@ -19,6 +19,7 @@
     'chpl_threads',
     'chpl_timers',
     'chpl_wide_pointers',
+    'utils',
```

Comment:
Is this needed? I personally have no idea, but it seems surprising if it is needed now, but wasn't previously since other util functions have been used.